src/components/button.js

```diff
@@ -0,0 +1,24 @@
+import PropTypes from 'prop-types';
+
+function Button({ name, className, color }) {
+  return (
+    <div className={className}>
+      <button type="button" className="w-100 btn btn-light " id={color}>
+        {name}
+      </button>
+    </div>
+  );
+}
+Button.propTypes = {
+  name: PropTypes.string,
+  className: PropTypes.string,
+  color: PropTypes.string,
+};
+
+Button.defaultProps = {
+  name: 'button',
```

Comment:
I guess it makes more sense to make the props name required Since in our case we do not want a button named `button` since there is no button with this name in a calculator.